src/components/molecules/Login.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useState } from "react";
+import { styled } from "@stitches/react";
+import { indigo, mauve, tomato } from "@radix-ui/colors";
+import * as TabsPrimitive from "@radix-ui/react-tabs";
+import { useForm, SubmitHandler } from "react-hook-form";
+import api from "@/lib/axios_settings";
+import Cookies from "js-cookie";
+
+type Props = {
+  setLoggedin: React.Dispatch<React.SetStateAction<boolean>>;
+};
+
+interface LoginFormInput {
+  email: string;
+  loginPassword: string;
+  confirmPassword: string;
+}
+
+const StyledContent = styled(TabsPrimitive.Content, {
+  flexGrow: 1,
+  padding: 20,
+  backgroundColor: "white",
+  borderBottomLeftRadius: 6,
+  borderBottomRightRadius: 6,
+  outline: "none",
+});
+
+// Exports
+export const TabsContent = StyledContent;
+
+const Flex = styled("div", { display: "flex" });
+
+const Text = styled("div", {
+  marginBottom: 20,
+  color: mauve.mauve11,
+  fontSize: 15,
+  lineHeight: 1.5,
+});
+
+const Button = styled("button", {
+  all: "unset",
+  display: "inline-flex",
+  alignItems: "center",
+  justifyContent: "center",
+  borderRadius: 4,
+  padding: "0 15px",
+  fontSize: 15,
+  lineHeight: 1,
+  fontWeight: 500,
+  height: 35,
+
+  variants: {
+    variant: {
+      indigo: {
+        backgroundColor: indigo.indigo4,
+        color: indigo.indigo11,
+        "&:hover": { backgroundColor: indigo.indigo5 },
+        "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo7}` },
+      },
+    },
+  },
+
+  defaultVariants: {
+    variant: "indigo",
+  },
+});
+const Fieldset = styled("fieldset", {
+  all: "unset",
+  marginBottom: 15,
+  width: "100%",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "flex-start",
+});
+
+const Label = styled("label", {
+  fontSize: 13,
+  lineHeight: 1,
+  marginBottom: 10,
+  color: indigo.indigo12,
+  display: "block",
+});
+
+const Input = styled("input", {
+  all: "unset",
+  flex: "1 0 auto",
+  borderRadius: 4,
+  padding: "0 10px",
+  fontSize: 15,
+  lineHeight: 1,
+  color: indigo.indigo11,
+  boxShadow: `0 0 0 1px ${indigo.indigo7}`,
+  height: 35,
+  "&:focus": { boxShadow: `0 0 0 2px ${indigo.indigo8}` },
+  "&::placeholder": { color: mauve.mauve7 },
+});
+
+const ErrorMsg = styled("p", {
+  color: tomato.tomato8,
+  fontSize: 12,
+  marginTop: 5,
+});
+
+export default function Login({ setLoggedin }: Props) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    getValues,
+  } = useForm<LoginFormInput>();
+  const [errMessage, setErrMessage] = useState<string | null>();
+  const onSubmit: SubmitHandler<LoginFormInput> = (data) => {
+    setErrMessage("");
+
+    const formData = new FormData();
+    formData.append("username", data.email);
+    formData.append("password", data.loginPassword);
+
+    api.post("/login", formData).then((res) => {
+      Cookies.set("access_token", res.data.access_token);
+      setLoggedin(true);
+    }).catch((error: any) => {
+      Cookies.remove("access_token");
```

Comment:
Do you need to remove the access token from cookie when the error occurs posting `/login`?
I think the `Cookies.remove` needs to be happen when the token expires or  at the `logout` feature.